Mathlib/Init/Algebra/Classes.lean

```diff
@@ -61,7 +61,7 @@ Mario made the following analysis of uses in mathlib3:
 
 universe u v
 
-class IsSymmOp (α : Type u) (β : outParam (Type v)) (op : outParam (α → α → β)) : Prop where
+class IsSymmOp (α : Type u) (β : Type v) (op : α → α → β) : Prop where
```

Comment:
This was obviously an error.  The whole point of unbundled classes is that we can have more than one operation per type, and the out-param version only allowed a single commutative operation per type.

The error was indirectly flagged by the new dangerousInstances linter: it complained about the `is_symm_op_of_is_commutative` instance below, correctly pointing out that it used to non-out-param `IsCommutative` class (which may have more than one solution for the operation✝) to (nondeterministically) fill in the out-param of the `IsSymmOp` class.

✝ And even worse, instances for `IsCommutative` may rely on `op` not being a metavariable.  This mismatch can result in nontermination of type class search.